src/Microsoft.Azure.WebJobs.Host/IConverterManager.cs

```diff
@@ -2,16 +2,14 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
-using System.Reflection;
-using Microsoft.Azure.WebJobs.Host.Bindings;
 
 namespace Microsoft.Azure.WebJobs
 {
     /// <summary>
     /// General service for converting between types for parameter bindings.  
     /// Parameter bindings call this to convert from user parameter types to underlying binding types. 
+    /// Converters can be added via extension configuration rules on <see cref="FluentConverterRules"/>.
     /// </summary>
-    [Obsolete("Not ready for public consumption.")]
     public interface IConverterManager
```

Comment:
>public interface IConverterManager [](start = 4, length = 34)

This public surface shrinks significantly. The "add" methods all now come through the Fluent binding rules.